osc/commands_git/pr_list.py

```diff
@@ -78,8 +84,20 @@ def run(self, args):
             if pr_obj_list:
                 total_entries += len(pr_obj_list)
                 pr_obj_list.sort()
-                for pr_obj in pr_obj_list:
-                    print(pr_obj.to_human_readable_string())
-                    print()
+                if not args.export:
+                    for pr_obj in pr_obj_list:
+                        print(pr_obj.to_human_readable_string())
+                        print()
+                else:
+                    sub_res = []
+                    for pr_obj in pr_obj_list:
+                        sub_res.append(pr_obj.dict(exclude_columns=["owner", "repo"]))
```

Comment:
Could you write a comment why we're excluding the 2 columns?
Also, it would be nice if you could come up with better variable names than `res_export` and `sub_res`.